client/src/main/java/com/boku/auth/http/client/BokuAPIClient.java

```diff
@@ -161,6 +161,7 @@ public RequestBuilder delete(String url) {
         private final HttpUriRequest request;
         private AuthorizationHeader authHeader;
         private String entityString;
+        private boolean requireSignedResponse = true;
```

Comment:
This should default to false, and then get set to true by `withAuthorization`.

(Or, if you'd rather not make the options order-sensitive, you could have this be null, and then handle the defaulting behavior inside execute(), but I think it's simpler to just do it here.)